### PR TITLE
Anchor clean-up

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by Youssef1313** [comment]
@mairaw, Can you re-open #15431 ?
It hasn't been fixed yet.

File: docs/framework/reflection-and-codedom/reflection-and-generic-types.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ ms.date: "03/30/2017"
 dev_langs: 
   - "csharp"
   - "vb"
+  - "cpp"
```
Comment (Youssef1313) [inline docs/framework/reflection-and-codedom/reflection-and-generic-types.md]: I think the cpp was missing.

**Review by Thraka — APPROVED** [review]
LGTM Thanks!

**Comment by mairaw** [comment]
Always double check if the PR has warnings! I'm going to take care of them.

**Pull request merged**